src/app/auth/reducer.js

```diff
@@ -77,6 +77,7 @@ export default (state = initialState, action) => {
       name: null,
       storageName: localStorage.getItem('name'),
       isOwner: false,
+      networkMatch: true,
```

Comment:
My justification for setting this to true and not rechecking is that the only way to log out is to be logged in originally, which requires networkMatch set to true.